script/golang.sh

```diff
@@ -11,10 +11,23 @@ fi
 DST_DIR=$SCRIPT_DIR/../pkg/
 # source directory for proto
 PROTO_DIR=$SCRIPT_DIR/../proto
+# Project directory is parent of script directory
+PROJECT_DIR="$(dirname "${SCRIPT_DIR}")"
+# Golang source location
+GO_SRC_DIR="${PROJECT_DIR}/go_build"
+# ensure the source directory exists
+mkdir -p ${GO_SRC_DIR}
+# remove any old source code laying around that may not get overwritten
+if [ -d ${GO_SRC_DIR} -a -w ${GO_SRC_DIR} ]; then
+  find ${GO_SRC_DIR} -type f -exec rm {} \;
+fi
 # pkg directory
 PKG_DIR=$SCRIPT_DIR/../proto/boundlessgeo_schema/
 # switch to the script dir in case of running the script ouside of this dir
 cd $SCRIPT_DIR
 # build protobufs
 protoc -I=$PROTO_DIR --go_out=plugins=grpc:$DST_DIR $PKG_DIR/*.proto
+#source generation script
+GEN_SRC_SCRIPT="${SCRIPT_DIR}/index.sh"
 
```

Comment:
remove this whitespace line, and expand the comment here to mention that this only builds the actions and events.json files for go